src/pages/LD_LeadConvertOverride.page

```diff
@@ -49,7 +30,7 @@
                 <div class="slds-p-horizontal_large">
 
                     <div class="slds-form-element slds-is-required">
-                        <c:UTIL_FormField sObjType="Contact" sObj="{!dummyCon}" field="OwnerId" labelOverride="{!$Label.leadConvertOwner}" />
+                        <c:UTIL_FormField sObjType="Contact" sObj="{!dummyCon}" field="OwnerId" labelOverride="{!$Label.leadConvertOwner}" overrideRequired="true"/>
```

Comment:
Is `UTIL_FormField` component setting fields as required by default when using new `npsp-common.css` and lightning stylesheet?